### DISCUSSION TITLE
发送数据包可能存在问题

### DISCUSSION THREAD

**Discussion by xiaxiangjun** [Tencent/tquic#68] [General]
![image](https://github.com/Tencent/tquic/assets/93415423/25d90d5c-5cac-4f74-b6da-c31cfb23d0b1)

实际测试中，当 done 为0时，之后就再也没有重试发送

**Comment by xiaxiangjun** [comment]
在应用层那边多调几次 `process_connections`方法就解决了

**Comment by wangfuyu** [comment]
> 在应用层那边多调几次 `process_connections`方法就解决了

这个要和endpoint.timeout()配合着用，不要直接多调用几次process_connections哈。
一次process_connections之后，调用一把endpoint.timeout()得到下次需要调用process_connections的时间。

**Reply by xiaxiangjun** [reply]
timeout有时返回  150多秒，这个超时感觉有点不准哟
